README.md

```diff
@@ -251,6 +251,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Mind](https://www.mind.pt/) | Information management and industrial systems. | `Lisboa` `Porto` |
 | [Moviik](https://www.moviik.com) | Customer Journey Management Platform. | `Braga` |
 | [Nmbrs](https://www.nmbrs.com/) [:rocket:](https://www.nmbrs.com/en/careers) [:octocat:](https://github.com/Nmbrs) | HR & Payroll software. | `Lisboa` |
+| [Omnibees](https://omnibees.com/) [:rocket:](https://omnibees-bee2pay-niara.gupy.io/) | Hotel Management Software. | `Faro` |
```

Comment:
```suggestion
| [Omnibees](https://omnibees.com/) [:rocket:](https://omnibees-bee2pay-niara.gupy.io/) | Hotel Management Software. | `Faro` `Remote` |
```
Do you think it also makes sense to have remote in the locations?

Reply:
I don't think so as they don't really do full remote. The best case scenario is 3 days a week WFH. You are forced to relocate no matter what basically.